pkgs/os-specific/darwin/apple-sdk/default.nix

```diff
@@ -201,6 +201,13 @@ in rec {
   };
 
   overrides = super: {
+    Accelerate = stdenv.lib.overrideDerivation super.Accelerate (drv: {
+      __propagatedImpureHostDeps = drv.__propagatedImpureHostDeps ++ [
```

Comment:
I don't think this is necessary since these paths are part of the Accelerate framework itself.